src/utils/arweave.ts

```diff
@@ -155,6 +243,37 @@ export const epochDistributionNoticeGqlQuery = ({
   });
   return gqlQuery;
 };
+// fallback query if the distribution notice does not get cranked
+export const epochDistributionNoticeGqlQueryFallback = ({
+  processId = ARIO_MAINNET_PROCESS_ID,
+  owners = ['OAb-n-ZugyN598kZNpfOy0ACelGVmwCQ0kYbgNGDUK8'], // ar.io team wallet ticks once a day
```

Comment:
this wallet is scheduled to tick once a day right at the distribution timestamp, if any other wallet ticks before it, then the query won't work...temporary solution